proposals-CG-WG.md

```diff
@@ -1,82 +1,78 @@
-# FedID CG/WG Process Mechanics
+# CG/WG Proposal Stages
+
+Author: @samuelgoto
+Created: 08/11/2024
+Last update: 09/11/2024
+Status: under review
 
 This is a proposal to break proposals into 5 stages of maturity, with clear guidelines and requirements to advance them:
 
-* [Stage 0](#stage0): Understand the Problem Space
-* [Stage 1](#stage1): Incubate Proposals
-* [Stage 2](#stage2): Specify a Preferred Proposal
-* [Stage 3](#stage3): Increase Deployment Confidence
-* [Stage 4](#stage4): Publish a Proposed Recommendation
+* [Stage 0](#stage0): **Exploration** of the Problem Space
+* [Stage 1](#stage1): **Incubation** of competing Alternatives
+* [Stage 2](#stage2): **Formalization** a preferred Proposal
+* [Stage 3](#stage3): **Independent Implementations** of the preferred Proposal
+* [Stage 4](#stage4): **Publication** of a Proposed Recommendation
+
```

Comment:
Suggest adding something like:
"These stages support the W3C process. If there is ever any question between W3C process requirements and how the groups will progress their work, the W3C process has precedent."

Reply:
sgtm done